phpunit.xml.dist

```diff
@@ -3,6 +3,7 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
     bootstrap="./vendor/autoload.php"
+    cacheResultFile="build/.phpunit.result.cache"
```

Comment:
Otherwise, the default location will be `.phpunit.result.cache`.

Since the `build` directory is already included in `.gitignore`, I think this makes sense.

Reply:
This works fine even when not using `phing`, see https://github.com/sebastianbergmann/phpunit/issues/3410.